plugin/restore/mysql/mysql_test.go

```diff
@@ -441,3 +441,28 @@ func TestParseAndSortBinlogFiles(t *testing.T) {
 		}
 	}
 }
+
+func TestBinlogFilesAreContinuous(t *testing.T) {
```

Comment:
test case for empty binlogFiles?

Reply:
Added.
However, it should not be reached.